src/test/java/com/thebois/models/world/structures/TownHallTests.java

```diff
@@ -0,0 +1,53 @@
+package com.thebois.models.world.structures;
+
+import org.junit.jupiter.api.Test;
+
+import com.thebois.models.Position;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class TownHallTests {
+
+    @Test
+    public void TownHallIsAlwaysCompleted() {
+        // Arrange
+        final IStructure house = StructureFactory.createStructure(
```

Comment:
Style
Might only be me, but I'm slightly bothered by the naming here, as you are testing TownHall but the variable is named house